website/developer-docs/setup/full-dev-environment.md

```diff
@@ -30,39 +30,71 @@ Depending on your platform, some native dependencies might be required. On macOS
 As long as [this issue](https://github.com/xmlsec/python-xmlsec/issues/252) about `libxmlsec-1.3.0` is open, a workaround is required to install a compatible version of `libxmlsec1` with brew, see [this comment](https://github.com/xmlsec/python-xmlsec/issues/254#issuecomment-1612005910).
 :::
 
-First, you need to create an isolated Python environment. To create the environment and install dependencies, run the following commands in the same directory as your authentik git repository:
+1. Create an isolated Python environment. To create the environment and install dependencies, run the following commands in the same directory as your authentik git repository:
 
 ```shell
 poetry shell # Creates a python virtualenv, and activates it in a new shell
-make install # Install all required dependencies  for Python and Javascript, including development dependencies
+make install # Installs all required dependencies for Python and Javascript, including development dependencies
 ```
 
-To configure authentik to use the local databases, we need a local config file. This file can be generated by running `make gen-dev-config`.
+2. Configure authentik to use the local databases using a local config file. To generate this file, run the following command in the same directory as your authentik git repository:
 
-To apply database migrations, run `make migrate`. This is needed after the initial setup, and whenever you fetch new source from upstream.
+```shell
+make gen-dev-config # Generates a local config file
+```
+
+To apply database migrations, run the following command in the same directory as your authentik git repository:
+
+```shell
+make migrate # Applies Database Migration
+```
+
+**NOTE**: This is needed after the initial setup and whenever you fetch new source from upstream.
```

Comment:
migrate is run automatically when running `ak server` so this entire block can be removed

Reply:
@BeryJu done!